crates/compiler/parse/src/ast.rs

```diff
@@ -1146,8 +1146,6 @@ impl<'a> ImportAlias<'a> {
 
 #[derive(Debug, Clone, PartialEq, Default)]
 pub struct Defs<'a> {
-    /// A collection of references by index to either `type_defs` or `value_defs`
-    /// It's an entry point for actual definitions, while `type_defs` and `value_defs` are append-only collections
```

Comment:
Can you share why this was removed?

Reply:
I added it earlier before when I had bad understanding of what's going on there. It has misleading information. It's better to write better documentation for this SOA as a whole or even refactor it. It currently combines two SOAs: tags and spaces

Reply:
I wrote documentation for it a couple of times tbh, but it looks really awkward anyway. I'll refactor it in a separate pr. This structure also deserves better dbg formatter